### PR TITLE
Add a althingi scraper and endpoint.

### PR DESCRIPTION
**Pull request by koddsson**
I think this is interesting data to a lot of people. @jamesrobb being one of those people ;)

### CODE REVIEW

**Comment by rbjarnason** [comment]
:+1:

**Comment by koddsson** [comment]
@benediktvaldez i fix

**Comment by koddsson** [comment]
Somebody pointed out that this needs tests. I'll add some later tonight.

**Comment by koddsson** [comment]
I added a test for the other method this PR adds but wasn't sure I should test anything else than 200 HTTP status code since the application code for scraping the website lives in the [althingi](https://www.npmjs.com/package/althingi) npm package.

**Comment by koddsson** [comment]
shit

**Comment by jamesrobb** [comment]
neato. I had actually been working on making my django module for this more versatile.

Question, I don't see to see where in the code you are actually scraping the xml, or referencing the xml address of where they keep this data stored. Am I missing something obvious here?

**Comment by MiniGod** [comment]
We've decided that for v2, all scraping will be done in separate node modules. There are a few reasons for this, but you can find the scraper here: https://www.npmjs.com/package/althingi

**Comment by jamesrobb** [comment]
Cool, thanks.

Mind my asking why you decided to separate it all out?

//-- James Robb
//-- +354.666.0887 (Iceland/Ísland)
//-- I am migrating away from my james@gearedinteractive.ca address
//-- Please send new communications to me at me@jamesrobb.ca

**Comment by MiniGod** [comment]
TLDR: Some people might want to use the scraper without calling apis.is. pre-alpha. Everything is open to suggestions and change.

------------

Version 2 is still in pre-alpha. Everything might change. We are honestly not 110% satisfied with this setup, but we did not want to have the whole project in the same repo. We wanted to separate concerns; the endpoints away from the core functionality.

We considered different ways to do this, either with submodules in git or with npm modules. We went with npm modules, but didn't want to have anything apis specific in them. This is because some people might want to run the scraper locally without calling apis.is.

Considering this, submodules in git would be a hassle as you would need to create a repo for the apis routing (probably like 10 lines or so) as well as the scraper.

We want the endpoint creation to be as simple as possible, while still not locking it down to only work on apis.
So... to create an endpoint, you first create (or find) a scraper, and publish it to npm. Then create the router in the apis project to use this scraper.

**Comment by jamesrobb** [comment]
Makes sense to me Kristján. Thanks!

//-- James Robb
//-- +354.666.0887 (Iceland/Ísland)
//-- I am migrating away from my james@gearedinteractive.ca address
//-- Please send new communications to me at me@jamesrobb.ca
